components/dropdown/dropdown-opener-mixin.js

```diff
@@ -172,6 +169,7 @@ export const DropdownOpenerMixin = superclass => class extends superclass {
 			return;
 		}
 		content.toggleOpen(applyFocus);
+		this._isOpen = !this._isOpen;
```

Comment:
This function isn't being used by the hover mixin, but I added this line in case a consumer wants to toggle the dropdown.